dipy/viz/horizon/tab/roi.py

```diff
@@ -1,52 +1,106 @@
+        self._tab_id = 0
 
-        opacity = 1
+        self._opacity_toggle = build_checkbox(
+            labels=[''],
+            checked_labels=[''],
+            on_change=self._toggle_opacity)
 
-        length = 450
-        lw = 3
-        radius = 8
-        fs = 16
-        tt = '{ratio:.0%}'
+        self._opacity_slider_label, self._opacity_slider = build_slider(
+            initial_value=1, max_value=1., text_template='{ratio:.0%}',
+            on_change=self._change_opacity, label='Opacity'
+        )
 
-        self.__slider_opacity = ui.LineSlider2D(
-            initial_value=opacity, max_value=1., length=length, line_width=lw,
-            outer_radius=radius, font_size=fs, text_template=tt)
+        self.register_elements(
+            self._opacity_toggle,
+            self._opacity_slider_label, self._opacity_slider
+        )
 
-        color_single_slider(self.__slider_opacity)
+    def _toggle_opacity(self, checkbox):
+        """Toggle opacity of all ROIs. Changing the slider.obj.value will
+        trigger the _change_opacity function and adjust the opacity.
 
-        self.__slider_opacity.on_change = self.__change_opacity
+        Parameters
+        ----------
+        checkbox : CheckBox2D
+        """
+        if '' in checkbox.checked_labels:
+            self._opacity_slider.selected_value = 1
+            self._opacity_slider.obj.value = 1
+        else:
+            self._opacity_slider.selected_value = 0
+            self._opacity_slider.obj.value = 0
 
-    def __change_opacity(self, slider):
+    def _change_opacity(self, slider):
+        """Change opacity of all ROIs.
+
+        Parameters
+        ----------
+        slider : LineSlider2D
+        """
         opacity = slider.value
-        for contour in self.__actors:
+        self._opacity_slider.selected_value = slider.value
+        for contour in self._actors:
             contour.GetProperty().SetOpacity(opacity)
 
-    def build(self, tab_id, tab_ui):
-        self.__tab_id = tab_id
-        self.__tab_ui = tab_ui
+    def build(self, tab_id, _tab_ui):
+        """Position the elements in the tab.
+
+        Parameters
+        ----------
+        tab_id : int
+        _tab_ui : TabUI
+            Object for Tab from FURY.
+        """
+
+        self._tab_id = tab_id
 
-        self.__tab_ui.add_element(
-            self.__tab_id, self.__slider_label_opacity, (.02, .85))
-        self.__tab_ui.add_element(
-            self.__tab_id, self.__slider_opacity, (.12, .85))
+        y_pos = .85
+        self._opacity_toggle.position = (.02, y_pos)
+        self._opacity_slider_label.position = (.05, y_pos)
+        self._opacity_slider.position = (.12, y_pos)
 
     @property
     def name(self):
-        return self.__name
+        """Title of the tab.
+
+        Returns
+        -------
+        str
+        """
+        return self._name
+
+    @property
+    def actors(self):
+        """Visualization actors controlled by tab.
```

Comment:
Visualization ? Strange description

Reply:
Yes it does not contain the UI actors but the actors you see in the visualization of the specific rois